slab/reedsolomon.go

```diff
@@ -90,6 +90,13 @@ func (s Slab) Reconstruct(shards [][]byte) error {
 
 // Recover recovers a slice of slab data from the supplied shards.
 func (s Slice) Recover(w io.Writer, shards [][]byte) error {
+	empty := true
```

Comment:
```go
empty := true
for _, s := range shards {
     if len(s) > 0 {
         empty = false
        break
     }
}
if empty {
     return nil
}
```

Reply:
4 lines is bad enough. Making it 7 lines is a travesty. This should be `shards.all(s => s == 0)` -- 1 line! -- but alas.